Tech Radar 2019.md

```diff
@@ -0,0 +1,39 @@
+At all times:
+
+- browser and hardware interoperation
+- accessibility
+- responsive and progressive content
+- longevity of content
+- user-centric privacy and security
+
+2019:
+
+- WebXR Device API
+- content creation tools like Blender integration of glTF
+- frameworks like Sumerian, PotassiumES
+
+2020:
+
+- payment integration
+- hand gesture input
+- eye tracking
+
+2021:
+
+- shared anchors
+- long lived, simultanous XR apps
```

Comment:
This seems too soon:  long lived, simultaneous apps imply things for the browser UIs (lots of things) which nobody is even thinking about yet.  2022 or longer, I'd say.